### PR TITLE
Fix total time statistic

### PR DESCRIPTION
**Pull request by nafur**
Since one of the last changes to the total time collection in the driver, this collection fails when the solver terminates unexpectedly (either by a signal or an exception). This PR fixes this issue and does some cleanup along the way.

### CODE REVIEW

File: src/main/main.h [hunk 1]
```diff
@@ -36,7 +38,19 @@ extern const char* progPath;
...
+
+/** Manages a custom timer for the total runtime in RAII-style. */
+class TotalTimer
+{
+    public:
+    TotalTimer(): d_start(std::chrono::steady_clock::now()) {}
+    ~TotalTimer();
+    private:
+    std::chrono::steady_clock::time_point d_start;
+};
```
Comment (mpreiner) [inline src/main/main.h:51]: clang-format

**Review by aniemetz — COMMENTED** [review]
LGTM but it seems you forgot to clang-format.

File: src/main/main.h (hunk 1 above)
Comment (nafur) [reply src/main/main.h]: done

File: src/main/signal_handlers.cpp [hunk 2]
```diff
@@ -21,6 +21,7 @@
 #include <string.h>
 
 #include <cerrno>
+#include <chrono>
```
Comment (mpreiner) [inline src/main/signal_handlers.cpp:24]: This include shouldn't be needed, right?

Comment (nafur) [reply src/main/signal_handlers.cpp]: You are right. I removed it.

**Review by mpreiner — APPROVED** [review]

**Pull request merged**